CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
   ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
+<<<<<<< HEAD
```

Comment:
Hey Dan,

Sorry for asking you to update things a lot.

It looks like you've brought your work up to date correctly but have left in the merge conflict syntax.

If you can remove the `<<<<<<< HEAD`, `=======` and `>>>>>>> Add classes to action items in the summary list component`

We'll be good to go.

Reply:
When you delete them if you use `git commit --amend` you can remove them without adding another commit too :)

Reply:
Sorry, my bad... I did update this but hadnt saved the file in my IDE before committing (bit of a school boy error)

it should be updated now